orchestration/templates/extract-from-json.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: extract-from-json
+spec:
+  templates:
+    - name: main
+      inputs:
+        parameters:
+          - name: pvc-name
+          - name: file-path
+          - name: json-key
+      volumes:
+        - name: state
+          persistentVolumeClaim:
+            claimName: '{{ "{{inputs.parameters.pvc-name}}" }}'
+            readOnly: true
+      {{- include "argo.retry" . | indent 6 }}
+      script:
+        image: python3-slim
+        command: [python]
+        source: |
+          python3 "scripts/extract-json-value.py" \
```

Comment:
This won't work as expected because `scripts/extract-json-value.py` exists on the local machine, not in the `python:3-slim` image. We either need to:
1. Write a `Dockerfile` that `ADD`s the script to the python base, then use that image here, or
2. Inject the contents of the script inline here as part of the multi-line string

I used approach 2 when I was writing the diffing template. The Helm blocks [like this](https://github.com/DataBiosphere/clinvar-ingest/blob/master/orchestration/templates/diff-table.yaml#L286-L288) are iterating through the lines of the script and rendering them into the WorkflowTemplate definition.

Reply:
You also don't need the `python3` prefix, since you have `python` in the `command` array above.